client/src/app/shared/components/sorting-list/sorting-list.component.html

```diff
@@ -0,0 +1,10 @@
+<cdk-drop class="list" (cdkDropDropped)="drop($event)">
+    <div class="box line" *ngFor="let item of array" cdkDrag>
+        <div class="section-one" cdkDragHandle>
```

Comment:
as discussed before, mobile users would _probably_ be confused why they could not scroll over the list itself.
make a TODO: comment so we remember to this later with some CSS magic. Having an area on the side where the user can scroll if enough for now.